### PR TITLE
Add a daily practice guide

### PR DESCRIPTION
**Pull request by nicholasjhenry**
Documents specifically how to check-in/check-out and highlights the
need to push code to GitHub at the end of the day

### CODE REVIEW

**Comment by nicholasjhenry** [comment]
@frahugo waiting for your review.

**Comment by nicholasjhenry** [comment]
@ChrisYammine and @tetiana12345678 to help standardize our check-ins/outs I've created a "daily practices" document. Could you please review.

File: guides/practices/daily.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+```
+out:
+• blocked: _Display 3 most recent tweets_; currently waiting on @john for review
+  https://github.com/montrealelixir/website/pull/13
+• completed: _Upgrade to Phoenix 1.3.x_; waiting review, paired with @jane
+  https://github.com/montrealelixir/website/pull/15
+• wip: _Display the next/most recent meetup from Meetup.com_; learned how the Meetup.com API works
+  https://github.com/montrealelixir/website/issues/12
+```
```
Comment (tetiana12345678) [inline guides/practices/daily.md]: We could take advantage of our signs here:
✔︎ completed: _Upgrade to Phoenix 1.3.x_; waiting review, paired with @jane
https://github.com/montrealelixir/website/pull/15
➽ wip: _Display the next/most recent meetup from Meetup.com_; learned how the Meetup.com API works
https://github.com/montrealelixir/website/issues/12

Comment (ChrisYammine) [reply guides/practices/daily.md]: seconding emojis/special characters as a legend -- quicker to parse especially when some bullets are multi-line

Comment (frahugo) [reply guides/practices/daily.md]: @ChrisYammine @tetiana12345678 Is there a keyboard/text shortcut for the emojis?  I'm not a fan of copying/pasting

**Comment by tetiana12345678** [comment]
@frahugo There is a way to add text shortcuts (on mac). I am not sure if we are all using mac...
screen shot 2017-11-13 at 12 17 15

The challenge is to think of rarely used combination and it supposed to be text apparently (so, using alt is not permitted in a replace column). Typing `~ch` would replace that text with the symbol (from my example). If replacement is not needed, typing second time the same text `~ch` will not replace the text with the symbol. 

I couldn't find a better way of setting a shortcut to the symbols. Hope that helps.

**Comment by nicholasjhenry** [comment]
Let's go with what we have documented here and we can revisit if we have the need/desire.

**Pull request merged**